### PR TITLE
Readme.md update

### PR DESCRIPTION
**Pull request by Saentist**
Add more information for project

### CODE REVIEW

**Comment by Flole998** [comment]
Could you please squash your commit together? Also there are changes in this which aren't Readme.md changes, those should be in a separate commit/PR.

File: README.md [hunk 1]
```diff
@@ -32,6 +37,8 @@ It supports the following outputs:
   * HTSP (own protocol)
   * SAT>IP
 
+Also can work with XMLTV EPG format
```
Comment (Flole998) [inline README.md:40]: "Also it can work with XMLTV EPG format" would probably be better here

Comment (Saentist) [reply README.md:40]: Acceptable, also with xmltv.sock

**Comment by Flole998** [comment]
Changes have been merged manually due to inactivity on this PR.

**Pull request closed without merging**